### PR TITLE
Snapshot for AboutUs and simple test for perseEuDate

### PR DESCRIPTION
**Pull request by Depesi**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/ita-social-projects/EventsExpress-Frontend/projects/1)

## Code reviewers
Vadym Kharyton
Andrii Kuzmichov
- [ ] @github_username
VadymKharyton
andrii-kuzmichov

## Summary of change
Create snapshot test for AboutUs component and simple test for parseEuDate.js

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/components/helpers/form-helpers/parseEuDate.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import parseEuDate from "./parseEuDate";
+
+it("Correct date", () => {
+  const result = parseEuDate("22-02-2002");
+  expect(result).toBe("2002-02-21T22:00:00.000Z");
+});
+
+describe("Uncorrect value", () => {
+  it("Don`t pass value", () => {
```
Comment (VadymKharyton) [inline src/components/helpers/form-helpers/parseEuDate.test.js]: Please give names for `tests` and for `describe` properly according to some conventions like [here](https://markus.oberlehner.net/blog/naming-your-unit-tests-it-should-vs-given-when-then/)

File: src/components/helpers/form-helpers/parseEuDate.test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import parseEuDate from "./parseEuDate";
+
+it("Correct date", () => {
```
Comment (VadymKharyton) [inline src/components/helpers/form-helpers/parseEuDate.test.js]: All tests should be wrapped with `describe` block. Please move it to existing one

File: src/components/AboutUs/AboutUs.test.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import React from "react";
+import { Provider } from "react-redux";
+import { BrowserRouter } from "react-router-dom";
+import renderer from "react-test-renderer";
+import configureStore from "redux-mock-store";
+import AboutUs from "./AboutUs";
+
+const mockStore = configureStore([]);
+
+it("renders when user don`t registered", () => {
```
Comment (VadymKharyton) [inline src/components/AboutUs/AboutUs.test.js]: Wrap tests with `describe` block

File: src/components/AboutUs/AboutUs.test.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  let store;
+  store = mockStore({
```
Comment (VadymKharyton) [inline src/components/AboutUs/AboutUs.test.js]: Create and initialize this variable in one line and with `const`.

File: src/components/AboutUs/AboutUs.test.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        ,
+      </BrowserRouter>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+it("renders when user registered", () => {
+  let store;
+  store = mockStore({
```
Comment (VadymKharyton) [inline src/components/AboutUs/AboutUs.test.js]: Create and initialize this variable in one line and with `const`.

Comment (Depesi) [reply src/components/AboutUs/AboutUs.test.js]: Done

File: src/components/AboutUs/AboutUs.test.js (hunk 4 above)
Comment (Depesi) [reply src/components/AboutUs/AboutUs.test.js]: Done

File: src/components/AboutUs/AboutUs.test.js (hunk 3 above)
Comment (Depesi) [reply src/components/AboutUs/AboutUs.test.js]: Done

File: src/components/helpers/form-helpers/parseEuDate.test.js (hunk 2 above)
Comment (Depesi) [reply src/components/helpers/form-helpers/parseEuDate.test.js]: Done

File: src/components/helpers/form-helpers/parseEuDate.test.js (hunk 1 above)
Comment (Depesi) [reply src/components/helpers/form-helpers/parseEuDate.test.js]: Done

**Review by VadymKharyton — APPROVED** [review]

**Pull request merged**